src/Appbar.tsx

```diff
@@ -61,6 +64,10 @@ function Appbar({ offline, loggedIn, username }: { offline: boolean; loggedIn: b
             <Typography variant="h6" color="inherit" component="div" sx={{ flexGrow: 1 }}>
                unbiased diversified team generator
             </Typography>
+            <Button sx={{ mt: 1}} variant="text" color="inherit" onClick={() => setDialogOpened(true)}>
```

Comment:
```suggestion
            <Button variant="text" color="inherit" onClick={() => setDialogOpened(true)}>
```
I believe this will make the margins for the button better.